apps/api-journeys/prisma/factories/nua9.ts

```diff
@@ -0,0 +1,1527 @@
+  const stepPrayer2 = await prisma.block.create({
+    data: {
+      journeyId: journey.id,
+      blockType: 'StepBlock',
+      extraAttrs: {
+        locked: false,
+        nextBlockId
+      },
+      parentOrder: 4
+    }
+  })
+  await prisma.block.create({
+    data: {
+      journeyId: journey.id,
+      blockType: 'ButtonBlock',
+      parentBlockId: prayerCard1.id,
+      extraAttrs: {
+        label: 'Next',
+        variant: 'contained',
+        color: 'primary',
+        size: 'small',
+        startIcon: {
+          name: 'PlayArrow'
+        },
+        action: {
+          gtmEventName: 'click',
+          blockId: stepPrayer2.id
+        }
+      },
+      parentOrder: 3
+    }
+  })
+  const prayerImageId2 = uuidv4()
+  const prayerCard2 = await prisma.block.create({
+    data: {
+      journeyId: journey.id,
+      blockType: 'CardBlock',
+      parentBlockId: stepPrayer2.id,
+      extraAttrs: {
+        themeMode: ThemeMode.dark,
+        themeName: ThemeName.base,
+        coverBlockId: prayerImageId2
+      },
+      parentOrder: 0
+    }
+  })
+  await prisma.block.create({
+    data: {
+      id: prayerImageId2,
+      journeyId: journey.id,
+      blockType: 'ImageBlock',
+      parentBlockId: prayerCard2.id,
+      extraAttrs: {
+        src: 'https://images.unsplash.com/photo-1504227488287-65981d97c2d6?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1170&q=80',
+        alt: 'Decision',
+        width: 1920,
+        height: 1080,
+        blurhash: 'L3D+P*1%00V]0H:%}+^NKHw?^0M|'
+      },
+      parentOrder: 0
+    }
+  })
+  await prisma.block.create({
+    data: {
+      journeyId: journey.id,
+      blockType: 'TypographyBlock',
+      parentBlockId: prayerCard2.id,
+      extraAttrs: {
+        content: '2/3',
+        variant: 'h6',
+        color: 'primary',
+        align: 'right'
+      },
+      parentOrder: 1
+    }
+  })
+  await prisma.block.create({
+    data: {
+      journeyId: journey.id,
+      blockType: 'TypographyBlock',
+      parentBlockId: prayerCard2.id,
+      extraAttrs: {
+        content:
+          'I confess, that I need your forgiveness. I want to turn from living for myself and trust you to take the lead in my life,',
+        variant: 'subtitle1',
+        color: 'primary',
+        align: 'left'
+      },
+      parentOrder: 1
+    }
+  })
+
+  // Third part of the prayer
+  const stepPrayer3 = await prisma.block.create({
+    data: {
+      journeyId: journey.id,
+      blockType: 'StepBlock',
+      extraAttrs: {
+        locked: false,
+        nextBlockId
+      },
+      parentOrder: 5
+    }
+  })
+  await prisma.block.create({
+    data: {
+      journeyId: journey.id,
+      blockType: 'ButtonBlock',
+      parentBlockId: prayerCard2.id,
+      extraAttrs: {
+        label: 'Next',
+        variant: 'contained',
+        color: 'primary',
+        size: 'small',
+        startIcon: {
+          name: 'PlayArrow'
```

Comment:
```suggestion
          name: 'ChevronRight'
```
When it's added into backend